### PR TITLE
tidb: add doc for global kill

### PR DESCRIPTION
**Pull request by Joyinqin**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add SQL & config docs for global kill

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/4565
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-kill.md [hunk 1]
```diff
@@ -51,6 +51,37 @@ Query OK, 0 rows affected (0.00 sec)
 * 按照设计，`KILL TIDB` 语句默认与 MySQL 不兼容。负载均衡器后面通常放有多个 TiDB 服务器，这种默认不兼容有助于防止在错误的 TiDB 服务器上终止连接。
 * `KILL TIDB` 语句是 TiDB 的扩展语法。如果正尝试终止的会话位于同一个 TiDB 服务器上，可在配置文件里设置 [`compatible-kill-query = true`](/tidb-configuration-file.md#compatible-kill-query)。
 
+## Global kill <span class="version-mark">从 v5.0.0 版本开始引入</span>
```
Comment (TomShawn) [inline sql-statements/sql-statement-kill.md:54]:
```suggestion
## Global Kill <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
```

File: sql-statements/sql-statement-kill.md [hunk 2]
```diff
@@ -51,6 +51,37 @@ Query OK, 0 rows affected (0.00 sec)
...
+
+从 v5.0 版本开始，TiDB 为 [global kill](https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md) 提供实验性支持。启用该语句后，每台 TiDB 服务器会确保连接 ID 全局唯一。可以向任何 TiDB 服务器发送 `KILL` 语句，该服务器将在内部请求路由到正确的 TiDB 实例。这样可以确保即使 TiDB 服务器位于负载均衡器之后，`KILL` 也是安全的。
```
Comment (TomShawn) [inline sql-statements/sql-statement-kill.md:56]:
```suggestion
从 v5.0 版本开始，TiDB 为 [Global Kill](https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md) 功能提供实验性支持。启用该功能后，连接到每台 TiDB 服务器的 ID 将是全局唯一的。客户端可以向任一 TiDB 服务器发送 `KILL` 语句，服务器再将请求路由至目标 TiDB 实例。即使 TiDB 服务器位于负载均衡器之后，`KILL` 也能安全地执行 。
```

File: sql-statements/sql-statement-kill.md [hunk 3]
```diff
@@ -51,6 +51,37 @@ Query OK, 0 rows affected (0.00 sec)
...
+
+要启用 `global kill` 功能，需要在配置文件的 `experimental` 中设置 `enable-global-kill = true`。
```
Comment (TomShawn) [inline sql-statements/sql-statement-kill.md:58]:
```suggestion
要启用 Global Kill 功能，需要在配置文件的 `experimental` 部分设置 `enable-global-kill = true`。
```

File: sql-statements/sql-statement-kill.md [hunk 4]
```diff
@@ -51,6 +51,37 @@ Query OK, 0 rows affected (0.00 sec)
...
+
+## Global kill 示例
+
+在 TiDB 实例 `127.0.0.1:10180` 上：
```
Comment (TomShawn) [inline sql-statements/sql-statement-kill.md:62]:
```suggestion
在 TiDB 实例 `127.0.0.1:10180` 上执行如下语句：
```

File: sql-statements/sql-statement-kill.md [hunk 5]
```diff
@@ -51,6 +51,37 @@ Query OK, 0 rows affected (0.00 sec)
...
+
+```sql
+SELECT SLEEP(60);
+```
+
+在 TiDB 实例 `127.0.0.1:10180` 上：
```
Comment (TomShawn) [inline sql-statements/sql-statement-kill.md:68]:
```suggestion
在 TiDB 实例 `127.0.0.1:10080` 上执行如下查询以及 `Kill` 操作：
```

File: tidb-configuration-file.md [hunk 6]
```diff
@@ -594,3 +594,8 @@ experimental 部分为 TiDB 实验功能相关的配置。该部分从 v3.1.0 
 
 + 用于控制是否能创建表达式索引。
 + 默认值：false
+
+### `enable-global-kill` <span class="version-mark">从 v5.0.0 版本开始引入</span>
```
Comment (TomShawn) [inline tidb-configuration-file.md:598]:
```suggestion
### `enable-global-kill` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
```

File: tidb-configuration-file.md [hunk 7]
```diff
@@ -594,3 +594,8 @@ experimental 部分为 TiDB 实验功能相关的配置。该部分从 v3.1.0 
...
+
++ 开启此功能后，即使 TiDB 服务器位于负载平衡器后面，也可以安全地终止任何连接。
```
Comment (TomShawn) [inline tidb-configuration-file.md:600]:
```suggestion
+ 开启 Global Kill 功能后，即使 TiDB 服务器位于负载均衡器后，也可以安全地终止任何连接。
```

File: sql-statements/sql-statement-kill.md [hunk 8]
```diff
@@ -51,6 +51,37 @@ Query OK, 0 rows affected (0.00 sec)
 * 按照设计，`KILL TIDB` 语句默认与 MySQL 不兼容。负载均衡器后面通常放有多个 TiDB 服务器，这种默认不兼容有助于防止在错误的 TiDB 服务器上终止连接。
 * `KILL TIDB` 语句是 TiDB 的扩展语法。如果正尝试终止的会话位于同一个 TiDB 服务器上，可在配置文件里设置 [`compatible-kill-query = true`](/tidb-configuration-file.md#compatible-kill-query)。
 
+## Global kill <span class="version-mark">从 v5.0.0 版本开始引入</span>
+
+从 v5.0 版本开始，TiDB 为 [global kill](https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md) 提供实验性支持。启用该语句后，每台 TiDB 服务器会确保连接 ID 全局唯一。可以向任何 TiDB 服务器发送 `KILL` 语句，该服务器将在内部请求路由到正确的 TiDB 实例。这样可以确保即使 TiDB 服务器位于负载均衡器之后，`KILL` 也是安全的。
+
+要启用 `global kill` 功能，需要在配置文件的 `experimental` 中设置 `enable-global-kill = true`。
+
+## Global kill 示例
```
Comment (TomShawn) [inline sql-statements/sql-statement-kill.md:60]:
```suggestion
## Global Kill 示例
```

File: sql-statements/sql-statement-kill.md [hunk 9]
```diff
@@ -51,6 +51,37 @@ Query OK, 0 rows affected (0.00 sec)
 * 按照设计，`KILL TIDB` 语句默认与 MySQL 不兼容。负载均衡器后面通常放有多个 TiDB 服务器，这种默认不兼容有助于防止在错误的 TiDB 服务器上终止连接。
 * `KILL TIDB` 语句是 TiDB 的扩展语法。如果正尝试终止的会话位于同一个 TiDB 服务器上，可在配置文件里设置 [`compatible-kill-query = true`](/tidb-configuration-file.md#compatible-kill-query)。
 
+## Global Kill <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
+
+从 v5.0 版本开始，TiDB 为 [Global Kill](https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md) 功能提供实验性支持。启用该功能后，连接到每台 TiDB 服务器的 ID 将是全局唯一的。客户端可以向任一 TiDB 服务器发送 `KILL` 语句，服务器再将请求路由至目标 TiDB 实例。即使 TiDB 服务器位于负载均衡器之后，`KILL` 也能安全地执行。
```
Comment (TomShawn) [inline sql-statements/sql-statement-kill.md:56]:
```suggestion
从 v5.0 版本开始，TiDB 为 [Global Kill](https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md) 功能提供实验性支持。该功能用于安全地终止客户端与 TiDB 服务器间的连接。启用该功能后，连接到每台 TiDB 服务器的 ID 将是全局唯一的。客户端可以向任一 TiDB 服务器发送 `KILL` 语句，服务器再将请求路由至目标 TiDB 实例。即使 TiDB 服务器位于负载均衡器之后，`KILL` 也能安全地被执行。
```

**Review by XuHuaiyu — COMMENTED** [review]
LGTM

File: tidb-configuration-file.md [hunk 10]
```diff
@@ -600,3 +600,8 @@ experimental 部分为 TiDB 实验功能相关的配置。该部分从 v3.1.0 
 
 + 用于控制是否能创建表达式索引。
 + 默认值：false
+
+### `enable-global-kill` <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
+
++ 开启 Global Kill 功能后，即使 TiDB 服务器位于负载均衡器后，也可以安全地终止任何连接。
```
Comment (TomShawn) [inline tidb-configuration-file.md:606]:
```suggestion
+ 用于控制是否开启 Global Kill 功能。将配置项的值设为 `true` 可开启该功能。开启后，即使 TiDB 服务器位于负载均衡器后，也可以安全地终止任何连接。
```

File: sql-statements/sql-statement-kill.md [hunk 11]
```diff
@@ -51,6 +51,37 @@ Query OK, 0 rows affected (0.00 sec)
 * 按照设计，`KILL TIDB` 语句默认与 MySQL 不兼容。负载均衡器后面通常放有多个 TiDB 服务器，这种默认不兼容有助于防止在错误的 TiDB 服务器上终止连接。
 * `KILL TIDB` 语句是 TiDB 的扩展语法。如果正尝试终止的会话位于同一个 TiDB 服务器上，可在配置文件里设置 [`compatible-kill-query = true`](/tidb-configuration-file.md#compatible-kill-query)。
 
+## Global Kill <span class="version-mark">从 v5.0.0-rc 版本开始引入</span>
+
+从 v5.0 版本开始，TiDB 为 [Global Kill](https://github.com/pingcap/tidb/blob/master/docs/design/2020-06-01-global-kill.md) 功能提供实验性支持。该功能用于安全地终止客户端与 TiDB 服务器间的连接。启用该功能后，连接到每台 TiDB 服务器的 ID 将是全局唯一的。客户端可以向任一 TiDB 服务器发送 `KILL` 语句，服务器再将请求路由至目标 TiDB 实例。即使 TiDB 服务器位于负载均衡器之后，`KILL` 也能安全地被执行。
+
+要启用 Global Kill 功能，需要在配置文件的 `experimental` 部分设置 `enable-global-kill = true`。
```
Comment (TomShawn) [inline sql-statements/sql-statement-kill.md:58]:
```suggestion
要启用 Global Kill 功能，需要在配置文件的 `experimental` 部分设置 [`enable-global-kill = true`](/tidb-configuration-file.md#enable-global-kill-从-v500-rc-版本开始引入)。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
@Joyinqin After this PR is merged, please help align the changes to the [English version](https://docs.pingcap.com/tidb/dev/sql-statement-kill) thanks!

**Comment by Joyinqin** [comment]
> @Joyinqin After this PR is merged, please help align the changes to the [English version](https://docs.pingcap.com/tidb/dev/sql-statement-kill) thanks!

Updated in https://github.com/pingcap/docs/pull/4729~

**Comment by pingyu** [comment]
LGTM. I'm sorry for the late.

**Pull request merged**